games-action/dave_gnukem/patches/dave_gnukem-1.0.patchset

```diff
@@ -0,0 +1,126 @@
+From c389e0fbedf93b329f8be312e7feeeef822694c7 Mon Sep 17 00:00:00 2001
+From: Abdur-Raheem Idowu <abdurraheemidowu@gmail.com>
+Date: Sat, 28 Dec 2019 13:39:31 +0000
+Subject: SDL include fix
+
+
+diff --git a/Makefile b/Makefile
+index ea9b75a..8fc1184 100644
+--- a/Makefile
++++ b/Makefile
+@@ -13,7 +13,7 @@ CC = gcc
+ 
+ 
+ # dj2016-10 Add L -I/usr/local/include/SDL in process of getting this working on Mac OS X - not sure if this is 'bad' to just have both /usr/include and /usr/local/include??
+-INCLUDEDIRS= -I/usr/include/SDL -I/usr/local/include/SDL 
++INCLUDEDIRS= $(shell sdl-config --cflags --libs)
+ 
+ #CCFLAGS = -O -Wall $(INCLUDEDIRS)
+ 
+-- 
+2.19.0
+
+
+From 4f03ac77591909627d4bfec5f4fc6062bd09acda Mon Sep 17 00:00:00 2001
+From: Abdur-Raheem Idowu <abdurraheemidowu@gmail.com>
+Date: Sat, 28 Dec 2019 13:45:45 +0000
+Subject: Fixed settings directory
+
+
+diff --git a/src/djstring.cpp b/src/djstring.cpp
+index 76237f9..2ba7187 100644
+--- a/src/djstring.cpp
++++ b/src/djstring.cpp
+@@ -22,6 +22,11 @@ Copyright (C) 1998-2018 David Joffe
+ #include <errno.h>//mkpath
+ #endif
+ 
++#ifdef __HAIKU__
++#include <Path.h>
++#include <FindDirectory.h>
++#endif
++
+ char *djStrDeepCopy( const char * src )
+ {
+ 	if (!src)
+@@ -284,6 +289,12 @@ std::string djGetFolderUserSettings()
+ #ifdef WIN32
+ 	std::string s = getenv("APPDATA");
+ 	djAppendPathS(s, "DaveGnukem");
++#elif __HAIKU__
++	std::string s;
++	BPath * s_path = new BPath();
++	find_directory(B_USER_SETTINGS_DIRECTORY, s_path);
++	djAppendPathS(s, s_path->Path());
++	djAppendPathS(s, ".gnukem");
```

Comment:
I would go for "gnukem" instead of ".gnukem", no need to created hidden files/directory, just nitpicking :)

Reply:
Maybe even "Duke Gnukem".

Reply:
Heh. Or "Dave Gnukem" ...:)

Reply:
Done!